test/clj/game_test/cards/identities.clj

```diff
@@ -614,7 +614,7 @@
     (prompt-select :corp (get-ice state :hq 0))
     (is (= 3 (:credit (get-corp))) "Corp not charged for Architects of Tomorrow rez of Eli 1.0")))
 
-(deftest haas-bioroid-asa-group
+(deftest haas-bioroid-asa-group-operation-install
```

Comment:
Small request: instead of multiple tests, nest all these tests under `(testing)` calls within the base-level `asa-group` deftest so testing against all Asa Group tests is one call instead of many. (Even smaller nitpick: the name of the ID is Asa Group, not Haas Bioroid: Asa Group, lol)